ghost/core/core/server/services/explore-ping/ExplorePingService.js

```
@@ -50,8 +50,12 @@ module.exports = class ExplorePingService {
             // Continue without member statistics
         }
 
+        console.log('this.config.get("url")', this.config.get('url'));
```

ESLint

[error] Unexpected console statement.

(no-console)